src/dispatcher/dev.js

```diff
@@ -0,0 +1,14 @@
+import pino from "pino";
+import pretty from 'pino-pretty';
+
+const logger = pino(
+  {},
+  pino.multistream(
+    [
+      pretty({ colorize: true }),
+      { stream: pino.destination(`./combined.log`) },
+    ]
+  )
+);
+
+logger.info('Asfdasdf')
```

Comment:
?

Reply:
good catch. I accidentally committed my dev script. I fixed it